tests/test_recipes.py

```diff
@@ -580,10 +582,10 @@ def test_list_with_repeat(self):
 
 
 class RandomPermutationTests(TestCase):
-    """Tests for ``random_permutation()``"""
+    """Tests for ``random_permutation()``."""
 
     def test_full_permutation(self):
-        """ensure every item from the iterable is returned in a new ordering
+        """Ensure every item from the iterable is returned in a new ordering.
```

Comment:
Please refer to this comment: https://github.com/more-itertools/more-itertools/pull/948/files#r1921578029 .